RevenueCatUI/Purchasing/Purchases+RevenueCatUI.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  Purchases+RevenueCatUI.swift
+//
+//  Created by Antonio Pallares on 13/6/25.
+
+import Foundation
+@_spi(Internal) import RevenueCat
+
+extension RevenueCat.Purchases {
+
+    /// Overrides the preferred locale for RevenueCatUI components.
+    /// - Parameter locale: A locale string in the format "language_region" (e.g., "en_US").
+    /// Use `nil` to remove the override and use the default user locale determined by the system.
+    ///
+    /// Setting this will affect the display of RevenueCat UI components, such as the Paywalls.
+    /// - Important: This method only takes effect after `Purchases` has been configured.
+    public static func overridePreferredUILocale(_ locale: String?) {
```

Comment:
This is probably out of scope of this PR, but I find counterintuitive that a `static` method has to be used after `configure`. I think the main reason is that this is a silent error. 

What do you think about using an assertion? So it crashes in DEBUG, but has no effect in RELEASE?

Reply:
Changed in https://github.com/RevenueCat/purchases-ios/pull/5292/commits/699c216e63c518bd2f0eb0d5c203ae29b550fc15